contrib/easyconfig-templates/2022a/Bundle-common-components.eb.tmpl

```diff
@@ -0,0 +1,52 @@
+# Template for a bundle of components from a common organisation and using the same easyblock on GCCcore 11.3.0
+easyblock = 'Bundle'
+
+name = 'SoftwareName'
+version = '0.0.0'
+
+homepage = 'https://www.domain.org'
+description = """
+Description in 80 chars long column
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('DependencyName', '0.0.0'),
+]
+
+default_easyblock = 'CMakeMake'
+default_component_specs = {
+    'source_urls': ['https://github.com/org/%(namelower)s/archive'],
+    'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+    'start_dir': '%(name)s-%(version)s',
+}
+
+components = [
+    ('component1', '0.0.0', {
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
+    }),
+    ('component2', '0.0.0', {
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
+    }),
+    ('component3', '0.0.0', {
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/comp2', 'bin/comp3', f'lib/libcomp1.{SHLIB_EXT}'],
+    'dirs': ['include/comp1', 'include/comp2', 'include/comp3'],
+}
+
+sanity_check_commands = [
+    "comp2 -h",
```

Comment:
I think it would be useful to include the pattern where you need to grab the result of stderror from the command, which is quite common.

Reply:
sure, added those in https://github.com/easybuilders/easybuild-easyconfigs/pull/21984/commits/c9c090497809baa09ea2e6725dc935b7c7bba397